astroid/interpreter/_import/spec.py

```diff
@@ -150,7 +175,16 @@ def find_module(
             if modname in sys.stdlib_module_names or (
                 processed and processed[0] in sys.stdlib_module_names
             ):
-                spec = importlib.util.find_spec(".".join((*processed, modname)))
+                try:
+                    with warnings.catch_warnings():
+                        warnings.filterwarnings("ignore", category=Warning)
+                        spec = importlib.util.find_spec(".".join((*processed, modname)))
+                except ValueError:
+                    # `find_spec` raises a ValueError for modules that are missing their `__spec__`
+                    # attributes. We don't really understand why, but this is likely caused by
+                    # us re-implementing the import behaviour but not correctly.
```

Comment:
I really have no clue. The only reason I know that we need to catch this is because it was part of the stack of the original issue reporter. I still don't really know how to reproduce this case.

Reply:
That's fine, but let's just remove the comment. The function is documented to raise ValueError, so let's just catch it without a comment.